packages/fxa-graphql-api/src/lib/datasources/authServer.ts

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { DataSource, DataSourceConfig } from 'apollo-datasource';
+import { Client } from 'fxa-js-client';
+import { Container } from 'typedi';
+
+import { Context } from '../server';
+import { fxAccountClientToken } from '../constants';
+
+function snakeToCamel(str: string) {
+  return str.replace(/(_\w)/g, (m: string) => m[1].toUpperCase());
+}
+
+/**
+ * Converts an object with keys that are possibly snake_cased to an
+ * object with keys that are camelCased.
+ *
+ * @param obj Object with string keys
+ */
+export function snakeToCamelObject(obj: { [key: string]: any }) {
+  return Object.fromEntries(Object.entries(obj).map(([k, v]) => [snakeToCamel(k), v]));
```

Comment:
Not a real issue, probably not even a nit so much as a preference but: I kind of hate doing this sort of normalization. I've been bitten in the past by mistakes / confusion around properties from upstream data sources. I might be missing it - but is there a test or a test with a fixture that looks like auth-server API response to exercise this property renaming?

Reply:
Threw a test in to verify the camelCase conversion.

Reply:
The auth-server has a substantial amount of response validation to ensure it returns this correctly. I'm not sure adding more validation to make sure our response validation works helps a lot.